config/initializers/subcensus_authorizer.rb

```diff
@@ -0,0 +1,26 @@
+# Decidim does not allow using more that one authorizer to one AuthorizationHandle
+# This code adds an additional validation to the used authorizer
+
```

Comment:
Some grammar fixes:

Decidim does not allow to use more than one ActionAuthorization inside an AuthorizationHandler.
This code adds an aditional validation to the AgeActionAuthorization to validate the subcensus.